README.md

```diff
@@ -252,7 +260,7 @@ AMQPS (SSL) port RabbitMQ address: [5671]:
 https port for the RabbitMQ management plugin: [15671]:
 INFO:rmq_setup.pyc:Starting rabbitmq server
 Warning: PID file not written; -detached was passed.
-INFO:rmq_setup.pyc:**Started rmq server at /home/vdev/rabbitmq_server/rabbitmq_server-3.9.7
+INFO:rmq_setup.pyc:**Started rmq server at /home/vdev/rabbitmq_server/rabbitmq_server-3.9.29
 INFO:requests.packages.urllib3.connectionpool:Starting new HTTP connection (1): localhost
```

Comment:
WE should be able to not have the requests show up at all...shouldn't we?

Reply:
We explicitly set urllib3.connectionpool to INFO level in main.py

Reply:
I think we should make it warn rather than info.

Reply:
Done